serverservice/consts.go

```diff
@@ -1,12 +1,30 @@
 package serverservice
 
 const (
-	// Serverservice attribute namespace for device vendor, model, serial attributes
+	// Serverservice attribute namespace for device vendor, model, serial attributes.
 	ServerAttributeNSVendor = "sh.hollow.alloy.server_vendor_attributes"
 
-	// Serverservice attribute namespace for the BMC address
+	// Serverservice attribute namespace for the BMC address.
 	ServerAttributeNSBmcAddress = "sh.hollow.bmc_info"
 
-	// Serverservice attribute namespace for firmware set labels
+	// Serverservice attribute namespace for firmware set labels.
 	FirmwareAttributeNSFirmwareSetLabels = "sh.hollow.firmware_set.labels"
+
+	// Serverservice attribute namespace for inband firmware information.
+	FirmwareVersionInbandNS = "sh.hollow.alloy.inband.firmware"
+
+	// Serverservice attribute namespace for outofband firmware information.
+	FirmwareVersionOutofbandNS = "sh.hollow.alloy.outofband.firmware"
+
+	// Serverservice attribute for installed firmware.
+	FirmwareVersionNSInstalledAttribute = "firmware.installed"
+
+	// Serverservice attribute namespace for inband component status information.
+	StatusInbandNS = "sh.hollow.alloy.inband.status"
```

Comment:
I have similar concerns as above here in terms of naming convention and how the `status.health` attribute relates.

Reply:
this const identifies the field within `data` to look for the health value, `status->health`
```json
  "versioned_attributes": [
        {
          "namespace": "sh.hollow.alloy.inband.status",
          "data": {
            "status": {
              "health": "OK"
            }
          },
```

Reply:
Same idea here. It feels like "how we actually tease the data out of the JSON" isn't something we should document, but rather something that we should provide via some sort of interface or something.

Reply:
Similar to my comment above, the string is used to form the HSS query to retrieve health information for a server.